asciidoc-confluence-publisher-converter/src/main/resources/org/sahli/asciidoc/confluence/publisher/converter/macro/JiraIssue.rb

```diff
@@ -0,0 +1,22 @@
+require 'asciidoctor'
+require 'asciidoctor/extensions'
+
+class JiraIssue < Asciidoctor::Extensions::InlineMacroProcessor
+  use_dsl
+
+  named :jira
+  name_positional_attributes 'server'
+
+  def process parent, target, attrs
+    key = target
+    jira_macro = %(
+    <ac:structured-macro ac:name="jira" ac:schema-version="1">
+      <ac:parameter ac:name="key">#{key}</ac:parameter>
+    )
```

Comment:
```suggestion
    jira_macro = %(<ac:structured-macro ac:name="jira" ac:schema-version="1">
      <ac:parameter ac:name="key">#{key}</ac:parameter>
    )
```

Including a new line here results in the Jira macro causing a line break in whatever text it has been used

Reply:
Good point. I'll fix